### PR TITLE
Fixes 182: change api errors to error objects

### PR DESCRIPTION
**Pull request by rverdile**
Accompanies this PR: https://github.com/content-services/content-sources-backend/pull/126

Updates the error handling here to work with the error structure defined in said PR.

### CODE REVIEW

**Comment by rverdile** [comment]
Not sure if I found everywhere that needs updating.

**Comment by jlsherrill** [comment]
https://issues.redhat.com/browse/HMSCONTENT-182

File: src/services/Content/ContentQueries.ts [hunk 1]
```diff
@@ -14,7 +14,7 @@ import {
   validateContentListItems,
   EditContentListItem,
   EditContentRequest,
-  getGpgKey,
+  getGpgKey, ErrorResponse
```
Comment (Andrewgdewar) [inline src/services/Content/ContentQueries.ts:17]: format

**Review by Andrewgdewar — APPROVED** [review]
Tested with backend PR, works well.

**Comment by rverdile** [comment]
I recommend not merging before this: https://github.com/content-services/content-sources-backend/pull/126

**Pull request merged**